### PR TITLE
[fetch] script and upstart config to use touchscreen vertically

### PR DESCRIPTION
**Pull request by 708yamaguchi**
This pull request is based on https://github.com/jsk-ros-pkg/jsk_robot/pull/1201 ([fetch] add monitor fixture to guard side of monitor)

In this pull request,
- I add script (`vertical-touchscreen.sh`) to change touchscreen orientation to vertical
- I update script (`install_upstart.sh`) to install upstart config file for `vertical-touchscreen.sh`.

These scripts worked on real fetch robot.
I will watch whether these scripts work correctly for a few days.

@knorth55 @sktometometo 
I cherry-pick the commit (https://github.com/jsk-ros-pkg/jsk_robot/commit/15c8743dc3cbac1d149004842c40c85e9f6c15b1) on your `fetch_ws` branch.
Then, I exec the following command:
```
roscd jsk_fetch_startup/config
./install_upstart.sh
```
The previous `jsk-fetch-startup.conf` is saved as `jsk-fetch-startup.conf.bk`

### CODE REVIEW

**Comment by knorth55** [comment]
did you run this script on Fetch15?
it seems there is no change about the screen.
I also want you to merge changes in `/etc/init/jsk-fetch-startup.conf`, which are not merged in `install_upstart.sh`

**Comment by 708yamaguchi** [comment]
> did you run this script on Fetch15?
> it seems there is no change about the screen.

After we run this script on Fetch15, we can use fetch monitor vertically like https://user-images.githubusercontent.com/19769486/74715671-44a99280-5270-11ea-9246-e462e3a99ee9.jpg

> I also want you to merge changes in /etc/init/jsk-fetch-startup.conf, which are not merged in install_upstart.sh

OK, I push additional commit to merge current `/etc/init/jsk-fetch-startup.conf`

By the way, do we need to merge other upstart services (such as `jsk-gdrive.conf` or `jsk-log-wifi.conf`) to `install_upstart.sh`?
I think this is needed in another pull request.

**Comment by 708yamaguchi** [comment]
> did you run this script on Fetch15?
> it seems there is no change about the screen.

If you run these commands in your PC, some variables need to be changed.

**Comment by knorth55** [comment]
I mean this script is not running on fetch15
Did you run this script on fetch15 and are you  testing it on Fetch15?


2020年2月18日(火) 14:10 Naoya Yamaguchi <notifications@github.com>:

**Comment by knorth55** [comment]
Ah, I misunderstood your PR.
I found out the vertical-screen upstart service now.

File: jsk_fetch_robot/jsk_fetch_startup/config/install_upstart.sh [hunk 1]
```diff
@@ -34,3 +35,23 @@ EOF"
...
+# See https://askubuntu.com/questions/507496/how-to-start-gui-application-with-upstart
+description \"Use vertical touchscreen\"
+
+start on runlevel [2345]
+stop on runlevel [016]
+
+script
+    sleep 30 # Wait for X programs to start
+    export DISPLAY=:0
+    exec su fetch -c \". ${current_prefix_path}/setup.bash && ${jsk_fetch_startup}/scripts/vertical-touchscreen.sh\"
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/config/install_upstart.sh]: it looks that it does not depend on ROS, but sould we need to source setup.bash?

Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch_startup/config/install_upstart.sh]: Thank you for your commend.
I remove unnecessary source.

**Comment by 708yamaguchi** [comment]
@k-okada 
I think the touchpad on the back of the fetch works well.
Could you please merge this?

**Comment by knorth55** [comment]
it is merged in #1389

**Pull request closed without merging**